test/e2e/config/operator.yaml

```diff
@@ -58,7 +58,7 @@ variables:
   HELM_EXTRA_VALUES_FOLDER: "/tmp"
 
   # Rancher Configuration
-  RANCHER_VERSION: "v2.10.2"
+  RANCHER_VERSION: "v2.11.0-rc7"
   RANCHER_ALPHA_VERSION: "v2.11.0-alpha8"
```

Comment:
BTW - we need to revise how e2e test is installing rancher. It is currently locked to alpha version here.